subseas_flex_fcst/layout.py

```diff
@@ -18,11 +23,23 @@
 IRI_BLUE = "rgb(25,57,138)"
 IRI_GRAY = "rgb(113,112,116)"
 LIGHT_GRAY = "#eeeeee"
+leadTime_init = "wk1"
+
+#Initialization for start date dropdown to get a list of start dates according to files available
+filesNameList = glob.glob(f'{DATA_PATH}/{CONFIG["forecast_mu_filePattern"]}')
+startDates = []
+for idx, i in enumerate(filesNameList):
+    startDate = re.search("\w{3}-\w{1,2}-\w{4}",filesNameList[idx])
+    startDatedt = datetime.strptime(startDate.group(),"%b-%d-%Y")
+    startDateStr = startDatedt.strftime("%b-%-d-%Y")
+    startDates.append(startDatedt)
+startDates = sorted(set(startDates))
+startDates = [i.strftime("%b-%-d-%Y") for i in startDates] #needs to have date with no zero padding to match the file path namesme
```

Comment:
This will probably help to convert to a function as well, so we can modulate when wanting to use this for other kind of forecasts. Can make that an issue for now.

Reply:
converted to issue.